lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -142,7 +142,12 @@ object NotificationHandler extends CohortHandler {
           requiredField(address.street, "Contact.OtherAddress.street")
         }
       postalCode = address.postalCode.getOrElse("")
-      country <- requiredField(address.country, "Contact.OtherAddress.country")
+      country <-
+        if (CohortSpec.isMembershipPriceRiseMonthlies(cohortSpec)) {
+          requiredField(address.country.fold(Some("United Kingdom"))(Some(_)), "Contact.OtherAddress.country")
```

Comment:
Does it make sense to treat UK as the default?  If this is the agreed business rule then it's fine.

Reply:
Yes it does, because that's where we have most of the subscriptions